### DISCUSSION TITLE
How to show form on a page

### DISCUSSION THREAD

**Discussion by mxhunter** [filamentphp/filament#1572] [Help]
Hi, 
I'm new with Filament and Livewire , How can I show a form on filament page. I want show a form for settings on the page loads 
created a page using artisan command php artisan make:filament-page Settings
and tried render method but it shows error , I don't see any documentation , please help me 
Thanks

**Comment by danharrin** [comment]
Are you using our form builder to make the form?

**Reply by mxhunter** [reply]
yes

**Comment by mxhunter** [comment]
Hi @danharrin 

I figured it out 
Page implements HasForms and use InteractsWithForms;
that shows form using getFormSchema method 

Thank you and it would be great if there is clarification on documentation 
Thanks again for the wonderful filament 👍
